ihmc-perception/src/main/java/us/ihmc/perception/realsense/RealsenseDeviceManager.java

```diff
@@ -116,7 +117,7 @@ public rs2_device createDevice(String serialNumberToFind)
             if (deviceRecommendedFirmwareVersion != null)
                LogTools.info("Realsense device recommended firmware version: {}", deviceRecommendedFirmwareVersion);
 
-            if (deviceSerialNumber.contains(serialNumberToFind.toLowerCase()))
+            if (serialNumberToFind == null || deviceSerialNumber.contains(serialNumberToFind.toLowerCase()))
```

Comment:
Serial number can be specified, or left null

Reply:
Would there ever be a case where you would want to do this?

Reply:
I think we'd want the serial numbers if we have multiple realsense devices so we can specify which one to connect to. This seems very likely with Alexander